src/components/dialogs/export-dialog.jsx

```diff
@@ -104,16 +132,21 @@ const ExportDialog = ({
                 selectedFormat
             );
             let suffix;
+            const urlSearchParams = new URLSearchParams();
             if (Object.keys(currentParameters).length > 0) {
-                const urlSearchParams = new URLSearchParams();
                 const jsoned = JSON.stringify(currentParameters);
                 urlSearchParams.append('formatParameters', jsoned);
-                // we have already as parameters, the tokens, so use '&' in stead of '?'
-                suffix = '&' + urlSearchParams.toString();
+            }
+            if (!isBlankOrEmpty(studyName)) {
+                urlSearchParams.append('studyName', studyName);
             }
 
+            // we have already as parameters, the access tokens, so use '&' instead of '?'
+            suffix = urlSearchParams.toString()
```

Comment:
Need to test with an empty urlSearchParams

Reply:
tested